### DISCUSSION TITLE
如何用边的功能实现类型钢笔工具的效果

### DISCUSSION THREAD

**Discussion by Momo23015258** [antvis/X6#1966] [Q&A]
如何用边的功能实现类型钢笔工具的效果

**Comment by NewByVector** [comment]
希望这个 [demo](https://x6.antv.vision/zh/examples/showcase/faq#path-editor) 能帮到你。

**Comment by Momo23015258** [comment]
画图功能已经实现了，参考了这个demo用的是节点和边拼接的方式，用router的vertices有点局限，拖动vertices节点的时候另外的vertices点也会跟着动，不能更改路线的走向




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "antvis/X6"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月7日(星期六) 下午2:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [antvis/X6] 如何用边的功能实现类型钢笔工具的效果 (Discussion #1966)





 
希望这个 demo 能帮到你。
